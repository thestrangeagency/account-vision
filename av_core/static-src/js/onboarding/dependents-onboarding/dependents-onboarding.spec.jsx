import React from 'react';
import { shallow } from 'enzyme';

import DependentsOnboarding from './dependents-onboarding';

jest.mock('../../services/onboarding-api-service');

test('it renders', () => {
  const wrapper = shallow(<DependentsOnboarding nextPage="" returnYear="" />);
  expect(wrapper).toMatchSnapshot();

  wrapper.setState({ choseYes: true });
  expect(wrapper).toMatchSnapshot();
});
