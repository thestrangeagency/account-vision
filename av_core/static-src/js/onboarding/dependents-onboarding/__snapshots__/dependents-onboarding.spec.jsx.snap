// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<p>
  Loading...
</p>
`;

exports[`it renders 2`] = `
<p>
  Loading...
</p>
`;
